src/applications/virtual-agent/components/webchat/WebChat.jsx

```diff
@@ -25,7 +25,7 @@ const WebChat = ({ token, WebChatFramework, apiSession }) => {
   const requireAuth = useSelector(
     state => state.featureToggles.virtualAgentAuth,
   );
-
+  let conversationId = '';
```

Comment:
I don't think we want to move this up from the original location. We want to send the actual convo ID to `StartConvoAndTrackUtterances` ?